lib/Dist/Zilla/Plugin/OurPkgVersion.pm

```diff
@@ -111,6 +132,32 @@ sub munge_file {
 	}
 	return;
 }
+
+sub _identify_version_value_token {
+	my $self= shift;
+	my $val_tok;
+	my @want= ('our', '$VERSION', '=', undef, ';');
```

Comment:
Please put a space between the variable names and the = on the LHS.

```perl
    my @want = ...
```

There are a few places in the `.pm` and the `.t` files where this should be updated.  I don't care so much about the files in `corpus`.